fv3core/docker/entrypoint.sh

```diff
@@ -14,14 +14,8 @@ if [[ -d /external/fv3gfs-wrapper ]]
 then
     CC=mpicc MPI=mpich make -C /external/fv3gfs-wrapper build
 fi
-for package in /external/*
-do
-    if [[ -f $package/setup.py ]]
-    then
-        echo "Setting up $package"
-        pip install -e "$package" -c /constraints.txt
-    fi
-done
+
+pip install -e /external/pace-util -e /external/stencils -c constraints.txt
```

Comment:
I don't know why it triggered in this PR, but without this the packages get installed in the wrong order.